static/src/javascripts/projects/common/modules/navigation/membership.js

```diff
@@ -12,12 +12,18 @@ import arrowRight from 'svgs/icon/arrow-right.svg';
 import type { Banner } from 'common/modules/ui/bannerPicker';
 import userPrefs from 'common/modules/user-prefs';
 
-const accountDataUpdateLink = accountDataUpdateWarningLink =>
-    `${config.get('page.idUrl')}/${
-        accountDataUpdateWarningLink === 'contribution'
-            ? 'contribution/recurring/edit'
-            : `${accountDataUpdateWarningLink}/edit`
-    }`;
+const accountDataUpdateLink = accountDataUpdateWarningLink => {
+    switch (accountDataUpdateWarningLink) {
+        case 'contribution':
+            return `${config.get('page.idUrl')}/contribution/recurring/edit`;
+        case 'membership':
+            return `https://manage.theguardian.com/payment/membership?INTCMP=BANNER`;
```

Comment:
Are we okay with this url?

Reply:
who's we? and do you mean it being prod only url? or not coming from config like the profile base url?

Reply:
Ideally it should go on the config (especially since it's gonna be used more in the future!) but the last time i looked up how to do this it was mildly painful

Reply:
re-tested in CODE